Userland/Libraries/LibPDF/Encoding.h

```diff
@@ -421,7 +421,7 @@
     V("θ", theta, 113)                                           \
     V("ϑ", theta1, 74)                                           \
     V("3", three, 51)                                            \
-    V("", trademarksans, 228)                                   \
+    V("", trademarksans, 228)                                 \
```

Comment:
Ooo now *these* are odd. Why did it decide to indent some of these macros with unicode characters less than the surrounding lines? counting bytes instead of grapheme clusters are we, clang-format?

Reply:
Those are all Private Use Area characters. AFAIK clang *should* respect character width, but I assume PUA ones are just ignored